perun/collect/trace/systemtap/engine.py

```diff
@@ -27,11 +27,11 @@
 
 import perun.utils as utils
 import perun.utils.metrics as metrics
-from perun.utils.helpers import SuppressedExceptions
 from perun.logic.locks import LockType, ResourceLock, get_active_locks_for
 from perun.utils.exceptions import (
     SystemTapStartupException,
     SystemTapScriptCompilationException,
+    SuppressedExceptions,
```

Comment:
Maybe an idea for a separate issue: we should be careful with using `SuppressedExceptions` in loops and hot paths because a new context manager is created each time the code is executed. This can lead to a serious slowdown compared to the built-in try/except.

Reply:
Can you create an issue?